pkgs/applications/radio/js8call/default.nix

```
@@ -0,0 +1,62 @@
+{ lib
+, stdenv
+, fetchFromBitbucket
+, wrapQtAppsHook
+, pkg-config
+, hamlib
+, libusb1
+, cmake
+, gfortran
+, fftw
+, fftwFloat
+, qtbase
+, qtmultimedia
+, qtserialport
+}:
+
+stdenv.mkDerivation rec {
+  pname = "js8call";
+  version = "2.2.0";
+
+  src = fetchFromBitbucket {
+    owner = "widefido";
+    repo = pname;
+    rev = "24677a00343c2f3cc53f6762466a4b11893094f8";
+    sha256 = "sha256-4yGxfG1sdOwOOEhoD664TW7rcajUQzRyH/kqTVMpop0=";
+  };
+
+  nativeBuildInputs = [ wrapQtAppsHook ];
+  buildInputs = [
+    pkg-config
+    hamlib
+    libusb1
+    cmake
+    gfortran
+    fftw
+    fftwFloat
+    qtbase
+    qtmultimedia
+    qtserialport
+  ];
+
+  prePatch = ''
+    substituteInPlace CMakeLists.txt \
+        --replace "/usr/share/applications" "$out/share/applications" \
+        --replace "/usr/share/pixmaps" "$out/share/pixmaps" \
+        --replace "/usr/bin/" "$out/bin"
+  '';
+
+  patches = [ ./cmake.patch ];
+
+  meta = with lib; {
+    description = "Weak-signal keyboard messaging for amateur radio.";
+    longDescription = ''
+      JS8Call is software using the JS8 Digital Mode providing weak signal
+      keyboard to keyboard messaging to Amateur Radio Operators.
+    '';
+    homepage = "http://js8call.com/";
+    license = with licenses; [ gpl3 ];
```

```suggestion
    license = licenses.gpl3;
```
gpl3Plus or gpl3Only?